Resources/Textures/Objects/Materials/materials.rsi/meta.json

```
@@ -69,6 +109,14 @@
     {
       "name": "pyrotton"
     },
+    {
+      "name": "pyrotten-inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "pyrotten-inhand-right",
```

```suggestion
      "name": "pyrotton-inhand-right",
```
rename the pngs as well